source/utils/unhandle.ts

```diff
@@ -0,0 +1,34 @@
+import EventEmitter from 'events';
+
+type Origin = EventEmitter;
+type Event = string | symbol;
+type Fn = (...args: any[]) => void;
```

Comment:
`Fn` need a more descriptive name. Same with the other places the name `fn` is used. And use `unknown`.

Reply:
TypeScript complains about that when using `origin.removeListener(event, fn)`...